build/Dockerfile

```diff
@@ -24,7 +22,8 @@ ARG USER_UID=1000
 ARG USER_GID=$USER_UID
 
 # Create the user
-RUN groupadd --gid $USER_GID $USERNAME \
+RUN apt-get update && apt-get install --no-install-recommends \
```

Comment:
Best practice here is also to remove the cache folder:  && rm -rf /var/lib/apt/lists/*

Reply:
suggested changes updated in docker file.